src/main/com/fulcrologic/fulcro/server/config.clj

```diff
@@ -102,8 +102,8 @@
   it dynamically as the configuration loads.
   "
   ([] (load-config! {}))
-  ([{:keys [config-path]}]
-   (let [defaults (load-edn-file! "config/defaults.edn")
+  ([{:keys [defaults-path config-path]}]
```

Comment:
You cannot overload arity...these are both 1-arg functions.

Reply:
Yes, I've been relying upon the `make dev` command which keeps the tests in the watch mode.

Reply:
ah, sorry, mis-read the diff